pydocx/export/image_resize.py

```diff
@@ -0,0 +1,165 @@
+# coding: utf-8
+from __future__ import (
+    absolute_import,
+    division,
+    print_function,
+    unicode_literals,
+)
+
+import base64
+from StringIO import StringIO
+from urlparse import urlparse
+import posixpath
+
+import requests
+
+from requests.exceptions import InvalidSchema
+
+from pydocx.util import uri
+from PIL import Image
```

Comment:
> However, as I was working on this, I struggled to think of a time when you wouldn't want the large images resized. 

I can't really think of one, either. I'm +1 on making this the default behavior, even.